### ISSUE TITLE
Sneak has no visual feedback, which is confusing (IIRC there's a PR/patch for this or something like it)

### ISSUE THREAD

**Issue by sapier** [minetest/minetest#2063]

**Comment by Calinou** [comment]
It is called sneaking and not crouching for a reason. Also, the third person animation doesn't crouch.

We don't *have* to have a crouching system, the current sneaking system is enough.

**Comment by ShadowNinja** [comment]
In that case, the sneak key shouldn't be a down arrow, it should be something like this: 
![sneaking](https://cloud.githubusercontent.com/assets/3399154/5671151/bc746de0-9753-11e4-8bbe-5490bc97eeb0.png)

**Comment by sapier** [comment]
why do we climb a ladder down when you press the sneak button? shouldn't we just climb more silent? ;-)

**Comment by paramat** [comment]
Also, the player model doesn't bend over.

**Comment by ShadowNinja** [comment]
@paramat: Why did you close this?
I'm not sure if the model bends over -- that shouldn't be hard to implement, but it's unrelated to this issue.

**Comment by sofar** [comment]
agreed this, and a swimming animation, are totally things we should consider adding to the player character animation set.

**Comment by paramat** [comment]
> Why did you close this?

See Calinou's comment. The player animation doesn't bend over as in MC, so the eye level is not lowered, so lowering the camera would be inconsistent.
If this was a feature request for the player animation to bend over, plus a lowering of camera it would be okay.

> I'm not sure if the model bends over -- that shouldn't be hard to implement, but it's unrelated to this issue.

It's related because if the player model doesn't crouch then lowering the camera level as feedback would be inconsistent. 'Sneak' doesn't necessarily mean bend over as in MC, it's just slow careful wlaking.

As Calinou says, we don't have to have a crouching animation and lowering of camera, it's not essential.

**Comment by ShadowNinja** [comment]
@paramat: This issue is actually reported by me.  Sapier just pasted it from a list of issues I sent him.
The player model doesn't necessarily have to have an animation for everything, although adding that would probably be a good idea.

**Comment by sapier** [comment]
Yes I assume I meant something like moving the camera down a little bit.

**Comment by C1ffisme** [comment]
If we don't intend on moving the camera down a bit, we could change the FOV of the player, but I personally prefer the view being lowered since higher FOV in Minecraft suggests speed or potion effects, and a lower FOV from sneaking would just be annoying.

**Comment by DS-Minetest** [comment]
Isn't this an issue for [minetest_game](https://github.com/minetest/minetest_game/blob/fe67ffda7ca4326c85f114bd02797f25348931f9/mods/player_api/api.lua#L97L137)?

**Comment by Ferk** [comment]
Alternatively, maybe some change in the HUD could work (like how in Skyrim the crosshair changes when sneaking), or add an offset to the wielded item so it's moved aside a bit while sneaking.

But imho, this is not very important. You have to consciously keep your finger pressed on the sneak key so it's hard for you to forget that you are in sneak mode, and sneaking only makes a difference when you actually move, the slower movement would be enough feedback.

**Comment by paramat** [comment]
Oops, some kind of feedback might be good, it doesn't necessarily have to be camera lowering.

**Comment by ThomasMonroe314** [comment]
@paramat maybe some sort of visual darkening to imply stealth?

**Comment by benrob0329** [comment]
I would suggest a vignette, but that might be annoying to some players. Perhaps a change in how the wielditem moves (holding it lower, not bobbing with each step).

Please also keep in mind that MC is one of few games we can look to for inspiration with a similar visual style, so adding a similar effect is not necessarily a bad thing. Sneak/crouch is a fairly common mechanic though, and is almost always represented by a slight lowering of the camera and a "ducking" animation for the playermodel.

**Comment by paramat** [comment]
I agree feedback is needed, a slight lowering of camera may actually be the simplest and most lightweight way, better than many other suggestions here.

**Comment by C1ffisme** [comment]
I'd prefer the camera lowering... If I were to bend my legs or body while standing, my view would go downwards. Pretty simple, realistic and logical feedback.

**Comment by jastevenson303** [comment]
Camera lowering is for crouching.  Change the cursor for visual feedback on
sneaking.

**Comment by ClobberXD** [comment]
Sneaking is synonymous to slightly bending and moving in a very restricted / controlled manner, generally to not be noticed. Keeping this in mind, lowering the camera **smoothly** when player is sneaking and not climbing/swimming seems to be the way to go...
